localization/languages/tr.json

```diff
@@ -267,11 +267,11 @@
     "passwordCaptureSavePassword": "% için parola kaydedilsin mi?",
     "passwordCaptureSave": "Kaydet",
     "passwordCaptureDontSave": "Kaydetme",
-    "passwordCaptureNeverSave": null, //missing translation
+    "passwordCaptureNeverSave": "Asla kaydetme", 
     /* Password viewer */
     "savedPasswordsHeading": "Kayıtlı Parolalar",
     "savedPasswordsEmpty": "Kaydedilmiş parola yok.",
-    "savedPasswordsNeverSavedLabel": null, //missing translation
+    "savedPasswordsNeverSavedLabel": "Asla kayıt edilmeyecek parolalar", 
```

Comment:
I think this might also need to be shorter - the English is just "never saved". It's used in settings > view saved passwords:

<img width="690" alt="Screen Shot 2021-09-12 at 10 38 23 PM" src="https://user-images.githubusercontent.com/10314059/133020373-14dcadf4-05e8-4922-9283-b29c8e4a1d47.png">

Sorry for not catching that earlier.

Reply:
Sentences can get longer while placing them. I'll update it with a better alternative. A direct translation of this sentence would not make sense. I have to do some localization.